snafu/benchmarks/coremarkpro/__init__.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# flake8: noqa
+"""Uperf benchmark"""
+from snafu.benchmarks.coremarkpro.coremarkpro import Coremarkpro
```

Comment:
This file is needed for benchmark-wrapper to recognize this folder. Doesn't seem to like having `-` in the name.